lib/Storage/LocalStorageClient.cpp

```diff
@@ -0,0 +1,133 @@
+/// \file LocalStorageClient.cpp
+/// \brief Implementation of StorageClient operations on the local filesystem
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Storage/LocalStorageClient.h"
+#include "revng/Support/PathList.h"
+
+namespace revng::detail {
```

Comment:
I used `detail` because I wanted to stress the fact that a "user" (in the sense of, anyone not touching the internals) should not explicitly instantiate any of the implementation classes

Reply:
Drop `detail`, move to private header as suggested elsewhere.

Reply:
Dropped